src/main/index.ts

```diff
@@ -23,6 +23,12 @@ import { appEventBus } from "../common/event-bus"
 import { extensionLoader } from "../extensions/extension-loader";
 import { extensionManager } from "../extensions/extension-manager";
 import { extensionsStore } from "../extensions/extensions-store";
+import { filesystemProvisionerStore } from "./extension-filesystem";
+import packageInfo from "../../package.json"
+
+if (isTestEnv) {
+  packageInfo.lens.extensions.push("example-extension")
```

Comment:
Isn't it could be included inside actual test instead?

Reply:
I actually want it loaded why in development mode as well because we should be testing to make sure **all** in-tree extensions are being built and run during normal development so that none of them lag behind (as the example has been)

Reply:
Maybe we should remove this example-extension at all since it's actually not participating in app work and there is separated repo for examples https://github.com/lensapp/lens-extension-samples ?

Reply:
Maybe... how are we making sure that that repo stays up to date?

Reply:
By updating used `@k8slens/extensions`? Anyway imho since we have that repo this is the place for samples.

Reply:
I don't understand "... that repo this is the place for samples". Do you mean that the https://github.com/lensapp/lens-extension-samples repo should be for our samples and this in-tree should just be removed? (If that is the case then I am for it).

Reply:
yes